NEMP/mods.json

```diff
@@ -4206,8 +4206,80 @@
         "function": "CheckForgeJson",
         "active": true,
         "forgejson": {
-            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/master/media/update.json",
-            "mcversion": "1.8.9"
+            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/version/media/update.json",
+            "mcversion": "1.9"
+        }
+    },
+    "BuildingBricks": {
+        "function": "CheckForgeJson",
+        "active": true,
+        "forgejson": {
+            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/version/media/update.json",
+            "mcversion": "1.9.4"
+        }
+    },
+    "HardModeTweaks": {
+        "function": "CheckForgeJson",
+        "active": true,
+        "forgejson": {
+            "url": "https://raw.githubusercontent.com/hea3ven/HardModeTweaks/version/media/update.json",
+            "mcversion": "1.9"
+        }
+    },
+    "HardModeTweaks": {
```

Comment:
https://github.com/NotEnoughMods/NotEnoughModPolling/blob/master/NotEnoughMods_Polling.py#L311-L314 for where the magic happens